bot/cogs/purge.py

```diff
@@ -0,0 +1,33 @@
+from discord import HTTPException
+from discord.ext import commands
+from common.bot_logger import get_logger
+from common.helper_functions import reply_and_delete
+
+logger = get_logger(__name__)
+
+
+class PurgeCog(commands.Cog, name="Purge command"):
+    def __init__(self, bot):
+        self.bot = bot
+        logger.info("Purge cog initialized")
+
+    @commands.has_permissions(manage_messages=True)
+    @commands.command(
+        help="Removes the amount of messages specified in the argument. Usage: purge [#messages]"
```

Comment:
```suggestion
        help="Removes the amount of messages specified in the argument."
```

The usage is already part of the help command